documentation/src/main/asciidoc/stories/assembly_wildfly_modules.adoc

```diff
@@ -16,7 +16,7 @@ endif::community[]
 //Downstream content
 ifdef::downstream[]
 In Red Hat JBoss EAP (EAP) 7.4 applications can directly handle the `infinispan` subsystem without the need to separately install {ProductName} modules.
```

Comment:
it was possible for legacy EAP versions as well, but not supported. 
Do we have a better wording?

Reply:
I changed the wording a bit. Let me know